README.md

```diff
@@ -147,3 +147,11 @@ Graph](https://github.com/rollkit/rollkit/blob/main/specs/src/specs/rollkit-depe
 ## Code of Conduct
 
 See our Code of Conduct [here](https://docs.celestia.org/community/coc).
+
+## Audits
+
+| Date       | Auditor                                       | Version                                                                             | Report                                                  |
+|------------|-----------------------------------------------|-------------------------------------------------------------------------------------|---------------------------------------------------------|
+| 2024/01/12  | [Informal Systems](https://informal.systems/) | [v13.0.0](https://github.com/rollkit/rollkit/releases/tag/v0.13.0)   | [informal-systems.pdf](specs/audit/informal-systems.pdf) |
+| 2024/01/10 | [Binary Builders](https://binary.builders/)   | [v13.0.0](https://github.com/rollkit/rollkit/releases/tag/v0.13.0) | [binary-builders.pdf](specs/audit/binary-builders.pdf)   |
```

Comment:
The Version listed here should be the git sha that was audited. So maybe change this to commit hash or something.